Source/RapyutaSimulationPlugins/Public/Core/RRROS2Utils.h

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
this purely ros node, publisher, subscribe utils should be in rclUE if necessary.